src/Raven.Server/Dashboard/Cluster/Notifications/ClusterOverviewNotificationSender.cs

```diff
@@ -0,0 +1,61 @@
+// -----------------------------------------------------------------------
+//  <copyright file="ClusterOverviewNotificationSender.cs" company="Hibernating Rhinos LTD">
+//      Copyright (c) Hibernating Rhinos LTD. All rights reserved.
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using System;
+using System.Threading;
+using System.Xml;
+using Raven.Client.Util;
+using Raven.Server.NotificationCenter;
+using Raven.Server.ServerWide;
+
+namespace Raven.Server.Dashboard.Cluster.Notifications
+{
+    public class ClusterOverviewNotificationSender : AbstractClusterDashboardNotificationSender
+    {
+        private readonly RavenServer _server;
+        // private readonly TimeSpan _defaultInterval = TimeSpan.FromMinutes(1);
+        private readonly TimeSpan _defaultInterval = TimeSpan.FromSeconds(15); // todo - how often ?
+
+        public ClusterOverviewNotificationSender(int widgetId, RavenServer server, ConnectedWatcher watcher, CancellationToken shutdown) : base(widgetId, watcher, shutdown)
+        {
+            _server = server;
+        }
+
+        protected override TimeSpan NotificationInterval => _defaultInterval;
+
+        protected override AbstractClusterDashboardNotification CreateNotification()
+        {
+            var testUpTime = (SystemTime.UtcNow - _server.Statistics.StartUpTime).ToString("c");
```

Comment:
?

Reply:
not sure we can't to convert that to string -- iirc it should handle dates correctly